server/datastore/mysql/mysql.go

```diff
@@ -283,8 +287,23 @@ func (ds *Datastore) writeChanLoop() {
 	}
 }
 
+var tracedDriverName string
+
+func init() {
+	var err error
+	tracedDriverName, err = otelsql.Register("mysql", semconv.DBSystemMySQL.Value.AsString())
+	apmsql.Register(tracedDriverName, &mysql.MySQLDriver{}, apmsql.WithDSNParser(apmmysql.ParseDSN))
```

Comment:
`tracedDriverName` is empty at this point, if this is expected, then I'd just use "" and explain why.

Reply:
I don't understand what you're saying here.

Reply:
Basically missing err check?
```
tracedDriverName, err = otelsql.Register("mysql", semconv.DBSystemMySQL.Value.AsString())
if err != nil {
  panic(err)
}  
apmsql.Register(tracedDriverName, &mysql.MySQLDriver{}, apmsql.WithDSNParser(apmmysql.ParseDSN))
```

Reply:
This has been resolved in latest code.